setup.sh

```diff
@@ -47,7 +47,7 @@ else
    fi
 fi
 
-pip install django==1.10.6 pillow grandalf django-prepared-query appdirs
+pip install django pillow grandalf django-prepared-query appdirs
```

Comment:
I had meant to do this update at some point, great to see you did it first.
Is there a need to generate migrations, since you have touched the model files ?

Reply:
I'll play around a bit, but generally, the `manage.py makemigrations; manage.py migrate;` statement in `setup.sh` should take care of that. I'll see if there is anything else that might want migration work re: 2.x.